modules/encoders/hash/sha1/secgen_local/local.rb

```diff
@@ -1,5 +1,6 @@
 #!/usr/bin/ruby
 require_relative '../../../../../lib/objects/local_hash_encoder.rb'
+require 'openssl'
```

Comment:
Why does this change sometimes use `require 'openssl'` and sometimes `require 'sha3'`?

The Gemfile has `sha3` added

Reply:
Digest-SHA3 (the previous library) was one of the few gems causing problems with the install process - most likely due to the fact that it was [last updated in 2015](https://rubygems.org/gems/digest-sha3/versions/1.1.0).The OpenSSL library has some support for SHA3, but it does not cover all the variations present within SecGen - hence why a replacement SHA3 gemfile was sourced.
For reference, here is a list of [all the Digests available in OpenSSL](https://ruby.github.io/openssl/OpenSSL/Digest.html).